client/src/helpers/strings/wise.ts

```diff
@@ -17,40 +17,37 @@ const strings: PlatformStrings = {
 
   // Notarizations Instructions
   NO_NOTARIZATIONS_ERROR: `
-    No Wise tag requests found.
-    Please follow instructions in the extension to notarize an existing Wise account tag in the account page.
+    No Wise tag proofs found.
+    Please follow instructions in the extension to prove an existing Wise account.
   `,
   NO_TRANSFER_NOTARIZATIONS_ERROR: `
-    No Wise transfer requests found.
-    Please follow instructions in the extension to notarize an existing Wise payment in the transactions page.
+    No Wise transfer proofs found.
+    Please follow instructions in the extension to prove an existing Wise payment.
   `,
 
   // New Registration
   REGISTRATION_INSTRUCTIONS: `
-    You must register in order to use ZKP2P. Registration requires a notarized message from Wise, which is used
-    to prove you own a Wise account. Your Wise ID is hashed to conceal your identity.
+    You must register in order to use ZKP2P. Click on Register above to initiate registration.
   `,
 
   // On Ramp Instructions
   PROOF_FORM_TITLE_SEND_INSTRUCTIONS: `
-    Provide the transaction alert email you received from Wise containing "You have done a UPI txn"
-    to complete the order. You can sign in with Google to pull the email or manually upload/paste
-    the email.
+    Generate a proof of payment using the ZKP2P Extension to complete the order.
   `,
 
   // New Deposit
   NEW_DEPOSIT_INSTRUCTIONS: `
-    Depositing requires registering a valid Wise multi currency id. Provide the USDC liquidity to deposit and
+    Depositing requires registering a valid Wise multi currency account id. Provide the USDC liquidity to deposit and
```

Comment:
Recommend renaming `multi currency id` altogether in the client. All strings that display that currently were development placeholders. Suggest maybe `Wise recipient id`?